### PR TITLE
FizzBuzz in Java...

### LINKED ISSUE #22: [JAVA] : FizzBuzz
Given an integer n, return a string array answer (1-indexed) where:

answer[i] == "FizzBuzz" if i is divisible by 3 and 5.
answer[i] == "Fizz" if i is divisible by 3.
answer[i] == "Buzz" if i is divisible by 5.
answer[i] == i (as a string) if none of the above conditions are true.
 

Example 1:
  Input: n = 3
  Output: ["1","2","Fizz"]

Example 2:
  Input: n = 5
  Output: ["1","2","Fizz","4","Buzz"]

Example 3:
  Input: n = 15
  Output: ["1","2","Fizz","4","Buzz","Fizz","7","8","Fizz","Buzz","11","Fizz","13","14","FizzBuzz"]

Implementation for this problem is required in Java. If interested, please find the boilerplate code in `java/array/FizzBuzz.java`

### PR DESCRIPTION
**Pull request by Dnyanesh-Bachhav**
Resolved #22 
Added FizzBuzz in java...

### CODE REVIEW

File: java/array/FizzBuzz.java [hunk 1]
```diff
@@ -29,12 +29,36 @@ Given an integer n, return a string array answer (1-indexed) where:
...
+        }
+        else if(i % 5 == 0)
+        {
+            result.add("Buzz");
+            continue;
+        }
+        else{
+        result.add(Integer.toString(i));
+        }     
+        }
```
Comment (ShubhKotnala) [inline java/array/FizzBuzz.java:61]: Please format the code, the logic looks good.

File: java/array/FizzBuzz.java [hunk 2]
```diff
@@ -29,12 +29,36 @@ Given an integer n, return a string array answer (1-indexed) where:
 
 public class FizzBuzz {
     public static void main(String[] args) {
-        System.out.println(fizzBuzz(1));
+        System.out.println(fizzBuzz(15));
     }
 
     public static List<String> fizzBuzz(int n) {
         //TODO: implement this method
```
Comment (ShubhKotnala) [inline java/array/FizzBuzz.java:36]: Can be removed now

**Review by ShubhKotnala — CHANGES_REQUESTED** [review]

File: java/array/FizzBuzz.java (hunk 1 above)
Comment (Dnyanesh-Bachhav) [reply java/array/FizzBuzz.java:61]: okk..

Comment (Dnyanesh-Bachhav) [reply java/array/FizzBuzz.java]: Hey, I have formatted a code please check it...

**Review by ShubhKotnala — APPROVED** [review]

**Pull request merged**